### ISSUE TITLE
Can't install on Ubuntu 17.10 from PPA

### ISSUE THREAD

**Issue by francogp** [ilikenwf/apt-fast#132]
Using ppa, the config files cancel the installation process because of errors in it.
`
Configurando apt-fast (1.8.3~144+git3a6bd77-0ubuntu1~ppa4) ...
Unescaped left brace in regex is illegal here in regex; marked by <-- HERE in m/\${ <-- HERE _MAXNUM}/ at /var/lib/dpkg/info/apt-fast.config line 212.
`

**Comment by fcastilloec** [comment]
+1

**Comment by fuelflo** [comment]
+1

**Comment by rlineros** [comment]
I also have the same problem when installing apt-fast via PPA in Ubuntu 17.10. Here it is the output:
```
Setting up apt-fast (1.8.3~144+git3a6bd77-0ubuntu1~ppa4) ...
Unescaped left brace in regex is illegal here in regex; marked by <-- HERE in m/\${ <-- HERE _MAXNUM}/ at /var/lib/dpkg/info/apt-fast.config line 212.
dpkg: error processing package apt-fast (--configure):
 subprocess installed post-installation script returned error exit status 255
Errors were encountered while processing:
 apt-fast
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

**Comment by necromuralist** [comment]
It'd be nice if this got fixed, but in case anyone is blocked by this -  you can open the `/var/lib/dpkg/info/apt-fast.config` file (as root) and add a backslash to line 212. i.e. change this:
```
if ($downloadcmd =~ /\${_MAXNUM}/){
```
To this:

```
if ($downloadcmd =~ /\$\{_MAXNUM}/){
```

(add a backslash between the `$` and the `{`). Then the install should work.

**Comment by rlineros** [comment]
Perfect. 
I confirm that by editing `/var/lib/dpkg/info/apt-fast.config` with the indications of @necromuralist the installation problem gets fixed.
Thanks.

**Comment by Lasall** [comment]
Duplicate of #102
